### PR TITLE
Extraction of data classes and enums with namespace modification

### PR DESCRIPTION
**Pull request by RichardVasquez**
This should hopefully help with issues #7 and #8

### CODE REVIEW

File: Runtime/KongregateWeb.cs [hunk 1]
```diff
@@ -8,514 +8,475 @@
...
+        public static void RequestItemList(string[] tags = null)
         {
             AssertIsReady();
-            return _instance._userId;
+            requestItemList
+                (
+                 tags != null
+                     ? JsonConvert.SerializeObject(tags)
+                     : null
+                );
```
Comment (randomPoison) [inline Runtime/KongregateWeb.cs]: Seems like the auto-formatting changed how this statement was laid out. Could you revert this change?

**Comment by RichardVasquez** [comment]
I've updated per your request.
